packages/fxa-auth-server/test/local/routes/devices-and-sessions.js

```diff
@@ -1230,7 +1263,10 @@ describe('/account/devices/invoke_command', () => {
       '/account/devices/invoke_command'
     );
 
-    return runTest(route, mockRequest).then(() => {
+    return runTest(route, mockRequest).then((response) => {
+      assert.ok(response.enqueued);
+      assert.ok(response.notified);
+      assert.isUndefined(response.notifyError);
```

Comment:
If we're going to return this data, let's test that we actually return the data we want...